### PR TITLE
feat: Coin ATM Radar API

### PR DESCRIPTION
**Pull request by Syn-McJ**
- CoinFlip replaced with Coin ATM Radar 
- Fixed README
- Expanded state map with Canadian states
- Some refactoring

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,10 +67,22 @@ from https://console.cloud.google.com/functions/list?authuser=0&project=dash-wal
...
+Or for the testnet mode:
+
+```
+gcloud pubsub topics publish explore-dash-sync-trigger --attribute="mode=testnet"
+```
+
+Or for the testnet mode:
+
+```
+gcloud pubsub topics publish dash-explore-sync-trigger --attribute="mode=testnet"
```
Comment (HashEngineering) [inline README.md:85]: The last line says `dash-explore-sync-trigger` -- should this be removed, or does it need other text above that says what it does?

**Review by HashEngineering — CHANGES_REQUESTED** [review]
There is one question about the README.

File: README.md (hunk 1 above)
Comment (Syn-McJ) [reply README.md]: Yes, you are right. Removed.

**Review by HashEngineering — APPROVED** [review]
Excellent

**Pull request merged**